gems/core/views.py

```diff
@@ -635,6 +635,31 @@ def get_day_in_last_month(self):
         today = self.get_today().replace(day=1) - timedelta(days=1)
         return today
 
+    def get_this_quarter(self):
+        quarter_1 = (1, 2, 3)
+        quarter_2 = (4, 5, 6)
+        quarter_3 = (7, 8, 9)
+        # quarter_4 = ("October", "November", "December")
+
+        today = self.get_today()
+
+        if today.month in quarter_1:
+            start = today.replace(month=1, day=1, hour=0, minute=0, second=0, microsecond=0)
+            end = today.replace(month=3, day=31, hour=23, minute=59, second=59, microsecond=999999)
+            return [start, end]
+        elif today in quarter_2:
+            start = today.replace(month=4, day=1, hour=0, minute=0, second=0, microsecond=0)
+            end = today.replace(month=6, day=31, hour=23, minute=59, second=59, microsecond=999999)
+            return [start, end]
+        elif today in quarter_3:
+            start = today.replace(month=7, day=1, hour=0, minute=0, second=0, microsecond=0)
+            end = today.replace(month=9, day=31, hour=23, minute=59, second=59, microsecond=999999)
+            return [start, end]
+        else:
+            start = today.replace(month=10, day=1, hour=0, minute=0, second=0, microsecond=0)
+            end = today.replace(month=12, day=31, hour=23, minute=59, second=59, microsecond=999999)
+            return [start, end]

```

Comment:
Since you're using the [`__range`](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#range) filter I think you should be able to just do `end = today.replace(month=1, day=1, year=start.year + 1)` since the SQL generated does a `between` query.  May be a bit simpler than the `hour`, `minute`, `second`, and `microsecond` everywhere.